incubator/hnc/api/v1alpha1/hierarchy_types.go

```diff
@@ -19,6 +19,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
Maybe add a comment here to say that:

- Any changes here need to also be reflected in the kubebuilder:validation:Enum comment, below.
- The meanings of all these constants are defined in the comment to `Condition.Code`, below.

Reply:
Done

Reply:
done

Reply:
done